digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/tierekisteri/TierekisteriTrafficSignAssetClient.scala

```diff
@@ -32,16 +32,17 @@ class TierekisteriTrafficSignAssetClient(trEndPoint: String, trEnable: Boolean,
     val roadPartNumber = convertToLong(getMandatoryFieldValue(data, trRoadPartNumber)).get
     val startMValue = convertToLong(getMandatoryFieldValue(data, trStartMValue)).get
     val track = convertToInt(getMandatoryFieldValue(data, trTrackCode)).map(Track.apply).getOrElse(Track.Unknown)
+    val wrongSideInformation = getFieldValue(data, trLIIKVAST)
```

Comment:
Maybe not so serious but i don't like 'wrong' side. Opposite side sounds better :)

Reply:
Let's switch this name too!